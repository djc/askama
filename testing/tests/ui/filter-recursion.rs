use askama::Template;

#[derive(Template)]
#[template(path = "filter-recursion.html")]
struct Filtered {
    s: &'static str,
}

fn main() {}
