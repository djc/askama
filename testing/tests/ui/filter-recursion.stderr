error: failed to parse template source
 --> testing/templates/filter-recursion.html:1:255
       "|A|AA|A|A|A|A|AA|A|A|A|A|AA|A|A|A|A|AA|A"...
 --> tests/ui/filter-recursion.rs:3:10
  |
3 | #[derive(Template)]
  |          ^^^^^^^^
  |
  = note: this error originates in the derive macro `Template` (in Nightly builds, run with -Z macro-backtrace for more info)
